### ISSUE TITLE
[Track 3 | Technical Blog] Runtime Upgrades in Substrate: A Technical Walkthrough

### ISSUE THREAD

**Issue by danielbui12** [openguild-labs/open-contribution-bounty#25]
### Description
Technical walkthrough of performing runtime upgrades in Substrate, including best practices and potential pitfalls

### Resources
- Bounty: $40
- Reference: https://docs.substrate.io/maintain/runtime-upgrades

**Comment by duonghb53** [comment]
Hi @danielbui12 & @chungquantin Can I get this one?

**Comment by chungquantin** [comment]
@duonghb53 Yep, why not? Feel free to write a markdown blog directly in this comment section or send a finished link to a hosted blog of your own to `#proof-of-work` channel in OpenGuild Discord.

**Comment by duonghb53** [comment]
@chungquantin I think it's a markdown file with PR, right?
Do you have form for this or I have to define it?

**Comment by chungquantin** [comment]
@duonghb53 You can write directly here. That's ok too if you don't have a blog site. Or the better way would be creating a PR to our main blog site because it will be transferred sooner or later. Example: https://github.com/openguild-labs/openguild/pull/40

**Comment by duonghb53** [comment]
@chungquantin btw, Can I use Vietnamese for this blog or only English?

**Comment by chungquantin** [comment]
@duonghb53 Yes Vietnamese is open, we don't restrict the blog post must be in English. But if it is possible, we highly recommend the language to be English for internationalization.

**Comment by chungquantin** [comment]
@duonghb53 Any progress on this?

**Comment by duonghb53** [comment]
@chungquantin Yes, In progress

**Comment by zaixincheng174-ai** [comment]
Submitted repository PR for this bounty: https://github.com/openguild-labs/open-contribution-bounty/pull/72

Scope:
- Track 3 technical blog on Substrate runtime upgrades
- storage migration and try-runtime validation coverage
- reviewer checklist for source accuracy, completeness, usefulness, and editorial quality

Validation completed:
- markdownlint passed
- all markdown reference URLs returned HTTP 200
- git diff whitespace check passed
- ASCII scan passed

Bounty payout details can be provided from zaixincheng174-ai on acceptance.
